test/powershell/Language/Parser/Parsing.Tests.ps1

```diff
@@ -302,3 +302,11 @@ Describe 'expressions parsing' -Tags "CI" {
 Describe 'Hash Expression parsing' -Tags "CI" {
     ShouldBeParseError '@{ a=1;b=2;c=3;' MissingEndCurlyBrace 2
 }
+
+Describe 'Unicode escape sequence parsing' -Tag "CI" {
+    ShouldBeParseError '"`u{110000}"' InvalidUnicodeEscapeSequenceValue 4 # error offset is "`u{>>1<<10000}"
+    ShouldBeParseError '"`u{1234567}"' UnicodeEscapeSequenceMissingTerminator 10 # error offset is "`u{123456>>7<<}"
+    ShouldBeParseError '"`u219z"' InvalidUnicodeEscapeSequence 1
+    ShouldBeParseError '"`u{219z"' UnicodeEscapeSequenceMissingTerminator 7 # error offset "`u{219>>z<<"
+    ShouldBeParseError '"`u{}"' InvalidBracketedUnicodeEscapeSequence 1
+}
```

Comment:
You need more tests that are missing the closing quote, e.g. ``"`u``, ``"`u{``, etc.
I find it handy to type in samples interactively, PSReadline tends to hit these cases automatically as you type.